sema/subyt/subyt.py

```
@@ -31,9 +31,9 @@ def __init__(
         *,
         template_name: str,
         template_folder: str,
-        source: str = None,
+        source: str | None = None,
         extra_sources: Dict[str, str] = {},
```

Ruff

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)